### PR TITLE
feat(*): switch to dep for go dependency management

### PR DESCRIPTION
**Pull request by adamreese**

### CODE REVIEW

**Comment by bacongobbler** [comment]
which one do you want merged first, this or #88?

**Comment by adamreese** [comment]
doesn't matter to me, I just got excited working on go dependencies YAY

**Comment by adamreese** [comment]
@bacongobbler rebased

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ VERSION            := $(shell git describe --tags --abbrev=0 2>/dev/null)
 LDFLAGS            :=
 
 # Test Repo is https://github.com/deis/empty-testbed
-TEST_REPO_COMMIT=589e15029e1e44dee48de4800daf1f78e64287c0
```
Comment (bacongobbler) [inline Makefile]: This was accidentally broken in the rebase

**Comment by bacongobbler** [comment]
Everything else looks good!

File: Makefile (hunk 1 above)
Comment (adamreese) [reply Makefile]: good catch

**Pull request merged**